java/tournament-scheduling/src/main/resources/META-INF/resources/app.js

```diff
@@ -335,8 +335,8 @@ function analyze() {
 function getScoreComponents(score) {
     let components = {hard: 0, medium: 0, soft: 0};
 
-    $.each([...score.matchAll(/(-?[0-9]+)(hard|medium|soft)/g)], (i, parts) => {
-        components[parts[2]] = parseInt(parts[1], 10);
+    $.each([...score.matchAll(/(-?\d*(\.\d+)?)(hard|medium|soft)/g)], (i, parts) => {
+        components[parts[3]] = parseFloat(parts[1], 10);
```

Comment:
I wonder if we should propagate this to all the other quickstarts.
This is the only one (for now) that has a float in there, but this will be easy to miss once another quickstart tries to copy-paste from school timetabling etc.